Updated wording around creating a secret (#1112)
vmware-tanzu/tanzu-dev-portal: content/guides/ci-cd/tekton-gs-p1.md

tekton-pipelines-controller-6d55778887-df59t   1/1     Running             0          13s
```

> Finally, in a couple of the examples, you'll be pushing container images to Docker Hub, so create a secret that can be used by Tekton to log in to Docker Hub. Substitute the placeholder values with your own:

Reviewer: ```suggestion
Finally, in a couple of the examples, you'll be pushing container images to Docker Hub. Create a secret that Tekton can use to log in to Docker Hub. Substitute the placeholder values with your own:
```
